Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -51,8 +51,8 @@ deriving BEq
 /-- Possible errors that text-based linters can report. -/
 -- We collect these in one inductive type to centralise error reporting.
 inductive StyleError where
-  /-- The bare string "Adaptation note" (or variants thereof): instead, the
-  #adaptation_note command should be used. -/
+  /-- The bare string "Adaptation note" (or variants thereof):
+  instead, the #adaptation_note command should be used. -/
```

Comment:
Ooh, sneaky! Is this maybe something we can lint against? A line (in comments) etc starting with `#adaptation_note` but not actually an invocation of the command?

Reply:
I think it's rare enough that we don't need to worry about it.

Reply:
Ok, I guess it's not really worth the effort.